homeassistant/components/deconz/__init__.py

```diff
@@ -141,7 +141,8 @@ def async_add_remote(sensors):
 
         Field is a string representing a specific device in deCONZ
         e.g. field='/lights/1/state'.
-        Entity_id can be used to retrieve the proper field.
+        Entity_id can be used to retrieve the proper device (when both
```

Comment:
List entity id first. extend field with how it can be used either by itself or with enitity id